components/buttons/button.css

```diff
@@ -0,0 +1,111 @@
+.btn{
+    
+    padding: 0.5rem 1rem;
+    margin: 1.25rem 0;
+    /* border-radius: 0.3rem; */
+    background-color: transparent;
+    /* border: none; */
+    border: 2px solid grey;
+    font-size: 1rem;
+    text-align: center;
+    width: 12rem;
+    /* outline: none; */
+    cursor: pointer;
+}
+.btn-solid{
+    background-color: var(--btn-color);
+}
+.btn-outline{
+    border: 2px solid #0dafa0;
+    color: #159673;
```

Comment:
move colors to a separate file where all common properties will reside

Reply:
ok done